app/views/cases/offender_sar/_case_details.html.slim

```diff
@@ -22,7 +22,7 @@
           = render partial: 'cases/sar/third_party_relationship', locals: { case_details: case_details }
           = render partial: 'cases/shared/requester_name', locals: { case_details: case_details }
           = render partial: 'cases/offender_sar/third_party_company_name', locals: { case_details: case_details }
-          = render partial: 'cases/offender_sar/third_party_reference', locals: { case_details: case_details }
+          = render partial: 'cases/offender_sar/requester_reference', locals: { case_details: case_details }
```

Comment:
Is there anything in the spec for the `_case_details_spec` that needs updating if we add new values here?

Reply:
Just thinking As this are optional fields should they shoe if they are empty?

Reply:
Excellent points, thanks! I've been assuming we'd review the view spec when we do the ticket for the show page, but maybe that's the wrong way to do it?

On the showing of empty fields, I asked Alex and this is by intent